### PR TITLE
Fix source maps

### LINKED ISSUE #281: @ethereum-waffle/provider breaks source maps
Hey 👋

I've received lots of complaints about Buidler lines numbers being off. 

I took some time to debug it, and the root of the problem is that `@ethereum-waffle/chai` loads `@ethereum-waffle/provider`, which in turn loads `ganache-core` and breaks the source maps.

I think I can work around this in Buidler, but I'm not sure how to fix it in Waffle v2.

BW, I think this shows how important my proposal #254 is.

### PR DESCRIPTION
**Pull request by vanruch**
This should fix #281
Looks like line numbers problem was caused by ganache requireing "source-map-support/register".
After removing those requires, line numbers in failed tests are correct, but it needs further testing if it doesn't break anything else

### CODE REVIEW

**Comment by marekkirejczyk** [comment]
@sz-piotr

File: waffle-provider/package.json [hunk 1]
```diff
@@ -35,9 +35,11 @@
     "node": ">=10.0"
   },
   "dependencies": {
+    "@ethereum-waffle/ens": "^3.0.2",
     "ethers": "^5.0.1",
     "ganache-core": "^2.10.2",
-    "@ethereum-waffle/ens": "^3.0.2"
+    "patch-package": "^6.2.2",
+    "postinstall-postinstall": "^2.1.0"
```
Comment (sz-piotr) [inline waffle-provider/package.json:42]: why this?

Comment (vanruch) [reply waffle-provider/package.json:42]: https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn

**Review by sz-piotr — APPROVED** [review]

**Comment by marekkirejczyk** [comment]
Released in [3.0.3](https://github.com/EthWorks/Waffle/releases/tag/3.0.3)

**Pull request merged**